src/react/atlascode/config/auth/AuthDialog.tsx

```diff
@@ -203,8 +203,8 @@ export const AuthDialog: React.FunctionComponent<AuthDialogProps> = memo(
             [],
         );
 
-        const registerUrl = useCallback(register(validateStartsWithProtocol), []);
-        const registerRequiredString = useCallback(register(validateRequiredString), []);
+        const registerUrl = register(validateStartsWithProtocol);
+        const registerRequiredString = register(validateRequiredString);
```

Comment:
Same here - there's a `useCallback` underneath it already, but all of this feels very sus